db/seeds.rb

```diff
@@ -5,3 +5,42 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+ user = User.find_or_create_by(email: 'admin@email.com') do |user|
+  user.password = 'password'
+  user.password_confirmation = 'password'
+  user.username = 'McAdmins'
+  user.admin = true
+end
+
+user.tv_series_viewings.find_or_create_by(
+  title: 'Brooklyn Nine Nine',
+  url: '/tmdb/tv_series?show_id=48891',
+  show_id: '48891',
+  started_at: 2.weeks.ago,
+  ended_at: nil
+)
+
+user.tv_series_viewings.find_or_create_by(
+  title: 'Star Trek: The Next Generation',
+  url: '/tmdb/tv_series?show_id=655',
+  show_id: '655',
+  started_at: '2023-04-01'.to_datetime,
+  ended_at: '2024-05-27'.to_datetime
+)
+
+user.tv_series_viewings.find_or_create_by(
+  title: 'DS9',
+  url: '/tmdb/tv_series?show_id=580',
+  show_id: '580',
+  started_at: '2021-06-13'.to_datetime,
+  ended_at: '2023-01-23'.to_datetime
+)
+
+user.tv_series_viewings.find_or_create_by(
+  title: 'Star Trek: The Next Generation',
+  url: '/tmdb/tv_series?show_id=655',
+  show_id: '655',
+  started_at: '2019-12-28'.to_datetime,
+  ended_at: '2001-06-12'.to_datetime
+)
```

Comment:
Is there a reason we don't have _any_ seeds in the app? I know having movies is weird because of the api, but I'm curious about other assets. I built these because they were useful to me.

Reply:
> Is there a reason we don't have any seeds in the app?

😱 i didn't know we didn't. thanks for adding some!